packages/swapper/src/types.ts

```diff
@@ -371,10 +373,15 @@ export type Swap = {
   buyAsset: Asset
   status: SwapStatus
   sellTxHash?: string
+  buyTxHash?: string
+  statusMessage?: string | [string, Polyglot.InterpolationOptions] | undefined
   sellAccountId: AccountId | undefined
+  receiveAddress: string | undefined
   swapperName: SwapperName
-  sellAmountCryptoBaseUnit: string
-  buyAmountCryptoBaseUnit: string
+  expectedSellAmountCryptoBaseUnit: string
```

Comment:
sell amount isn't expected, it's only the buy amount which is an expected out

Reply:
Changed to sellAmountCryptoBaseUnit in [b1b4b95](https://github.com/shapeshift/web/pull/9695/commits/b1b4b95c981b3622adb720d3bc424bbed409656e)